src/partials/hero.html

```diff
@@ -14,4 +14,6 @@ <h2 class="title">Dungeons & Dragons: Honor Among Thieves</h2>
             <button class="details-btn">More details</button>
         </div>
     </div>
+
+    <script src="/src/js/hero.js"></script>
```

Comment:
Tutaj masz błąd ścieżki
Najlepiej usuń to i podepnij pod main.js
import './hero';